app/controllers/foreman_acd/concerns/app_instance_mixins.rb

```diff
@@ -22,7 +22,7 @@ def collect_host_report_data(app_instance)
             a_host.update({
                             :id => foreman_host.host.id,
                             :build => foreman_host.host.build,
-                            :hostUrl => host_path(foreman_host.host),
+                            :hostUrl => helpers.current_host_details_path(foreman_host.host),
```

Comment:
Is it possible to add a test for that?

Reply:
Also , does this modify our dependency to foreman, as in this method can only be used in foreman >= X.Y